src/main/java/pl/wrapper/parking/infrastructure/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,63 @@
+package pl.wrapper.parking.infrastructure.exception;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import pl.wrapper.parking.infrastructure.error.ErrorWrapper;
+
+@ControllerAdvice
+@RequiredArgsConstructor
+class GlobalExceptionHandler{
```

Comment:
why is there I_AM_A_TEAPOT status on every method? Change the status in error wrapper to match to status of responseEntity